ppr-ui/src/utils/feature-flags.ts

```diff
@@ -10,7 +10,7 @@ export const defaultFlagSet: LDFlagSet = {
   'search-registration-number': false,
   'search-serial-number': false,
   'mhr-ui-enabled': false,
-  'mhr-registration-enabled': false
+  'mhr-registration-enabled': true
```

Comment:
Are we enabling the MHR reg or it got committed by mistake?

Reply:
Definitely a mistake, good catch, thank you! 
Will revert

Reply:
👍🏻